docs/installing-oggm.rst

```diff
@@ -137,27 +142,18 @@ Install the packages from the `conda-forge`_ channel::
     recommend to **always** use the the `conda-forge`_ channel for your
     installation.
 
-You might consider setting `conda-forge`_ per default, as suggested on their
+You might consider setting `conda-forge`_ (and oggm) per default, as suggested on their
 documentation page::
 
     conda config --add channels conda-forge
+    conda config --add channels oggm
     conda install <package-name>
 
-If you want progress reports when downloading data files you can optionally
-install progressbar2::
-
-    pip install progressbar2
-
 No scientific python installation is complete without installing
 `ipython`_ and `jupyter`_::
 
     conda install -c conda-forge ipython jupyter
 
-After success, install the following packages from Fabien's github::
-
-    pip install git+https://github.com/fmaussion/motionless.git
```

Comment:
It's unrelated to this PR, but did you update your builds to use the newly released package on PyPi? It's no longer necessary to use my fork for install